networkx/classes/graphviews.py

```diff
@@ -133,7 +133,6 @@ def generic_graph_view(G, create_using=None):
     return newG
 
 
-@deprecate_positional_args(version="3.4")
```

Comment:
Should we also remove the vendored deprecate_positional_args too https://github.com/networkx/networkx/blob/0659f0b77857979d0af15841fd55115551095756/networkx/utils/decorators.py#L1241 ? I don't think we are using it anywhere else?